retrofit/src/main/java/retrofit2/http/Field.java

```diff
@@ -30,21 +30,22 @@
  * field pair for each non-{@code null} item.
  * <p>
  * Simple Example:
- * <pre>
+ * <pre>{@code
```

Comment:
Hah. Do you know why? I'm inclined to believe you since Javadoc is ridiculous, but I checked the IDEA-rendered and HTML-rendered Javadoc and what's in this PR rendered correctly.